code/go/0chain.net/smartcontract/faucetsc/handler.go

```diff
@@ -1,24 +1,30 @@
 package faucetsc
 
 import (
+	"0chain.net/smartcontract"
 	"context"
 	"fmt"
 	"time"
 	// "encoding/json"
 	"net/url"
 
 	c_state "0chain.net/chaincore/chain/state"
-	"0chain.net/core/common"
+)
+
+const (
+	noLimitsMsg     = "can't get limits"
+	noGlobalNodeMsg = "can't get global node"
+	noClient        = "can't get client"
 )
```

Comment:
what you think isn't it better to replace those with `errors`. So it will be easier to mentain those errors not only http handlers wide but also system wide like 
```
ErrNoLimitsMsg = errors.New("can't get limits")
```

this make it way easy to handle specific error using standard tools
also we are loosing original cause of error, maybe wrapping will be better

Reply:
This Pr does not seem to be creating any new packages, so don't understand why package naming is an issue here. 
Current error handling across `0chain` products is crude, with inline string constants being the norm. Might be an idea to create an issue to discuss error handling.

Reply:
@Sriep 
Sorry maybe I was not cleat, but I mean as @bbist mentioned and I totally agree with this. Ad this relates to any other application domain object, each error need to be defined in corresponding domain object or level of it's usage.

> Should we define smartcontract specific errors inside corresponding smartcontract package?

And for some reason some errors have have been moved to `common` package, we should avoid any new code in common directory.